config/initializers/services_initializer.rb

```diff
@@ -1,6 +1,7 @@
 unless Rails.env.test?
   BugzillaService.credentials = Settings.bugzilla_credentials
   GithubService.credentials   = Settings.github_credentials
+  PivotalService.credentials  = Settings.pivotal_credentials
 
-  PivotalService.credentials = Settings.pivotal_credentials
+  GithubService.logger = Sidekiq::Logging.logger
```

Comment:
Ah I suppose it makes sense to add rate limit logging to the sidekiq log wherein the workers who are doing the calling are doing the work :D derp.